cpp/tests/utilities_tests/column_wrapper_tests.cu

```diff
@@ -30,6 +32,17 @@ struct FixedWidthColumnWrapperTest : public cudf::test::BaseFixture,
   auto data_type() { return cudf::data_type{cudf::experimental::type_to_id<T>()}; }
 };
 
+struct ListColumnWrapperTest : public cudf::test::BaseFixture {
+};
+template <typename T>
+struct ListColumnWrapperTestTyped : public cudf::test::BaseFixture {
+  ListColumnWrapperTestTyped() {}
+
+  auto data_type() { return cudf::data_type{cudf::experimental::type_to_id<T>()}; }
+};
+using NumericTypesNoBool = cudf::test::Types<int8_t, int16_t, int32_t, int64_t, float, double>;
```

Comment:
Why not test bool?
Why not test timestamps?
Why not just use the FixedWidthTypes list? (That way it will automatically add duration tests and unsigned int tests)

On the other hand, the logic is not different for any of the fixed width types, right? So it might be sufficient to just test ints and then test strings and other nested types.

Reply:
The tests are using values > 1 so you get errors about type narrowing if bool is included in the list.   You're right about the timestamps though - I'll add them.

Reply:
If possible, it would be better to use one of the built-in type lists so that when we add new types (like unsigned ints, decimal32, etc.), they will get tested.

Reply:
Yep.  Updated it to be a concatenation of `IntegralTypes`, `FloatingPointTypes` and `TimestampTypes`